backend/core/types/src/backend-swagger.ts

```diff
@@ -3893,6 +3900,12 @@ export interface User {
 
   /**  */
   hitConfirmationURL?: Date
+
+  /**  */
+  activeAccessToken?: string
```

Comment:
@ludtkemorgan When I hit `/listings` and view the items under `leasingAgents`, I see this property show up along with some other potentially sensitive information.  My understanding of what this represents is limited to what I'm able to grok in bits and pieces in this PR, but at first look it appears as though I would be able to grab the current access and refresh tokens of a leasing agent if they are currently logged in.  Is that accurate?

Reply:
![image](https://user-images.githubusercontent.com/23032560/232921618-5a576244-b2da-4eba-ae59-5b470aa3a856.png)

Reply:
Confirmed:

![image](https://user-images.githubusercontent.com/23032560/232929179-7b2f2f64-ea13-49a8-84bd-eafd435dc86f.png)

This is a hard block until it's fixed on the Bloom side, IMO

Reply:
That is a good call out. I brought it up with the team today and we are going to put up another commit to core to remove the sensitive fields. There is a chance we can remove a majority if not all of the leasingAgent fields from the /listings endpoint

Reply:
Thanks for fixing and merging so quickly!